### ISSUE TITLE
Border inside is broken after today's update

### ISSUE THREAD

**Issue by dom1torii** [catppuccin/zen-browser#12]
Border that should be inside doesn't have catppuccin accent anymore (you can see it's dark on the screenshot)
Don't have the before screenshot sadly but you can see it here https://raw.githubusercontent.com/catppuccin/zen-browser/refs/heads/main/assets/mocha.webp
![image](https://github.com/user-attachments/assets/e48e9afe-3182-4569-a7c7-0f248014a9c7)

**Comment by dom1torii** [comment]
Also i just noticed, shouldn't address bar have an accent colored border too? I didn't have it since i installed the catppuccin theme 5 days ago

**Comment by IAmJafeth** [comment]
Hi @dom1torii,

Thanks for the report, this seems to be a duplicate of: https://github.com/catppuccin/zen-browser/issues/11

Please refer to the comments on that issue.

TLDR, with this update Zen changed the UI and will no longer use those accent borders, the border will still be present for split tabs.

Closing as duplicate: #11

**Comment by dom1torii** [comment]
> Hey @dom1torii ,
> 
> Just found out you can use the [Super Url Barl Zen Mod](https://zen-browser.app/mods/d93e67f8-e5e1-401e-9b82-f9d5bab231e6) to bring back the border around the URL bar in case you want to use it
> 
> ![image](https://private-user-images.githubusercontent.com/45522320/385564271-571eb4db-212d-4ce1-9506-96a9610ff349.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0OTU4MTYsIm5iZiI6MTczMTQ5NTUxNiwicGF0aCI6Ii80NTUyMjMyMC8zODU1NjQyNzEtNTcxZWI0ZGItMjEyZC00Y2UxLTk1MDYtOTZhOTYxMGZmMzQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEzVDEwNTgzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1NGY4OGE5NmEwZjZjMTg5NjdhYWVlMDNkNDYwODBiZGM4MDUwZjUyMTA0ODRlYjFhZDY0NjBhYWRkYmNiYWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.orLnkWVPzjeAJdswgJ3T-LhqJTKDvFRQN6aRwlQUgUo)

Thanks. It does.
I decided to search for some mods too and turns out [Zen Paper Cut Theme](https://zen-browser.app/mods/753c7518-237d-480a-abfd-1f42a7eabacc) can be used to bring back inner border, but only the top one sadly.
![image](https://github.com/user-attachments/assets/1426bdf8-7c81-4b98-b7be-6f275556efb5)
